app/views/hiring_staff/vacancies/documents/_title.html.haml

```diff
@@ -0,0 +1,12 @@
+- if @vacancy.published?
+  - content_for :page_title_prefix, "Edit the Supporting documents"
```

Comment:
Shouldn't Supporting be lower case or documents be title case?

Reply:
I agree. I just followed convention for all the other page titles. Happy to change them all 👍🏼

Reply:
It would be nice to I18n these strings

Reply:
If you're following convention then don't sweat it.  Let's get it shipped and refactor for casing/I18N later.